client/src/types/material-ui.d.ts

```diff
@@ -0,0 +1,15 @@
+import { Theme, ThemeOptions } from "@material-ui/core/styles/createMuiTheme";
+
+declare module "@material-ui/core/styles/createMuiTheme" {
+  interface Theme {
+    custom: {
+      maxContentWidth: string;
+    }
+  }
+  // allow configuration using `createMuiTheme`
+  interface ThemeOptions {
+    custom?: {
+        maxContentWidth?: string;
+    };
+  }
+}
```

Comment:
hmm, i feel like this shouldn't be necessary
what made you do this and was there a resource that told you to do it this way?

Reply:
I believe it was here: https://stackoverflow.com/questions/61120760/how-to-extend-material-ui-theme-with-typescript
In order to create a custom variable it has to be declared in Typescript or it will throw an error.

Reply:
Oh here it is: https://www.mi4t.me/blog/howt-to-extend-material-ui-theme/

Reply:
TypeScript complains if you don't declare the variable like this